lotus-soup/compositions/composition-drand-halt.toml

```diff
@@ -12,12 +12,12 @@
 [global.build]
   selectors = ["testground"]
 
-[global.build_config]
-  enable_go_build_cache = true
-
 [global.run_config]
   exposed_ports = { pprof = "6060", node_rpc = "1234", miner_rpc = "2345" }
 
+[global.build_config]
+  enable_go_build_cache = true
+
```

Comment:
uber nit: I usually like to specify build/run in chronological order: build first, run second.

Reply:
I changed it so that all compositions are consistent. and the build_config changes between runners.